composer/callbacks/checkpoint_saver.py

```diff
@@ -29,7 +29,7 @@
 _TORCH_DISTRIBUTED_CHECKPOINTS_METADATA_FILENAME = '.metadata'
 
 
-def checkpoint_periodically(interval: Union[str, int, Time]) -> Callable[[State, Event], bool]:
+def checkpoint_periodically(interval: Union[str, int, Time], save_end_of_training=True) -> Callable[[State, Event], bool]:
```

Comment:
We could rename and move this to a callbacks util file because it is used both by the CheckpointSaver and Generate callbacks